packages/EasyAsync/src/Implementations/Doctrine/DBAL/AbstractPersister.php

```diff
@@ -29,14 +29,14 @@ abstract class AbstractPersister
     protected $table;
 
     /**
-     * @var \EonX\EasyAsync\Interfaces\UuidGeneratorInterface
+     * @var \EonX\EasyRandom\Interfaces\UuidV4GeneratorInterface
      */
     protected $uuid;
 
     public function __construct(
         Connection $conn,
         DateTimeGeneratorInterface $datetime,
-        UuidGeneratorInterface $uuid,
+        UuidV4GeneratorInterface $uuid,
```

Comment:
Because we shouldn't explicitly register the uuid v4 generator in the service container we won't be able to use dependency injection on this interface, we could create a definition for it and use the `RandomGeneratorInterface` service to get the uuid v4 generator but I think it's more work for not much benefit.

So here you should inject `RandomGeneratorInterface` directly and use the `uuidV4()` method on it.

Reply:
Done.